### DISCUSSION TITLE
Trying to get nvim-cmp back

### DISCUSSION THREAD

**Discussion by Muhammad95959** [LazyVim/LazyVim#5282] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/bd74ca98-e57f-4a8a-9f58-a40b0926ee12)

nvim-cmp is disabled and the rest of cmp-* plugins aren't there
Am I doing something wrong?

**Comment by dpetka2001** [comment, marked as answer]
If you enable `nvim-cmp` from your `lazy.lua` file using the `import` feature, you also have to set in your `options.lua` the following `vim.g.lazyvim_cmp = "nvim-cmp"`. Also, the order of the `import`s **does matter**. 

If you instead enable the Extras via `:LazyExtras`, then this is taken care of for you. It's recommended for users to use the `LazyExtras` command to enable whatever Extra they want.
